js/docx/smoke/api_uni_color/get_class_type.js

```diff
@@ -4,6 +4,8 @@ var oGs1, oGs2, oFill, oStroke, oDrawing, oParagraph, oPresetColor;
 oParagraph = oDocument.GetElement(0);
 oPresetColor = Api.CreatePresetColor("lightYellow");
 oGs1 = Api.CreateGradientStop(oPresetColor, 0);
+// Using an RGBColor instance is valid for testing UniColor.
```

Comment:
I think the edits for the api_uni_color/get_class_type.js test are not related to the edits for the to_json tests

You can't interfere with third-party edits in a commit with the same logic (in our case to_json tests)

Reply:
fix in new commit